### PR TITLE
Expose cvm Ins and Outs

### PR DESCRIPTION
**Pull request by tasinco**
Dress the transaction with the c-chain addresses..

### CODE REVIEW

**Review by tyler-smith — APPROVED** [review]
Edit: Commented on wrong PR.

File: services/indexes/models/avax.go [hunk 1]
```diff
@@ -124,3 +127,18 @@ type OutputList struct {
 	ListMetadata
 	Outputs []*Output `json:"outputs"`
 }
+
+type CvmOutput struct {
```
Comment (tyler-smith) [inline services/indexes/models/avax.go:131]: What is the reason for creating a separate Output struct? At least c chain import inputs and c chain export outputs are the same type of Outpoint in the protocol as any other x or p chain  outpoint.

File: services/indexes/models/types.go [hunk 2]
```diff
@@ -3,7 +3,14 @@
 
 package models
 
+type CChainType uint16
```
Comment (tyler-smith) [inline services/indexes/models/types.go:6]: Same as the model bit with these types, why not make them part of the existing transaction and output types?

**Review by tyler-smith — CHANGES_REQUESTED** [review]
Some questions to discuss.

**Comment by tasinco** [comment]
we'll do https://github.com/ava-labs/ortelius/pull/228 instead.

**Pull request closed without merging**